Jenkinsfiles/cbc-pod-deployment-config-w-selenium-p311.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: v1
```

Comment:
Same here, about the 3.11-specific files

Reply:
we are updating with new image created in ECR and needed new file for 3.11